crates/bevy_ecs/src/schedule/executor_parallel.rs

```diff
@@ -56,7 +55,7 @@ pub struct ParallelExecutor {
 
 impl Default for ParallelExecutor {
     fn default() -> Self {
-        let (finish_sender, finish_receiver) = async_channel::unbounded();
+        let (finish_sender, finish_receiver) = async_channel::bounded(128);
```

Comment:
This needs a comment capturing the writeup from your PR comment.

Reply:
Done.